impl/src/main/java/org/jboss/weld/context/beanstore/AttributeBeanStore.java

```diff
@@ -104,6 +101,20 @@ public boolean attach() {
         }
     }
 
+    /**
+     * Read all relevant attributes from the backing store and copy instances which are not present in the local bean store.
+     */
+    public void readAttributes() {
```

Comment:
What about `loadUninitializedAttributes()`? The `attributes` part follows the convention used for other similar methods (getAttribute(), getAttributeNames()) and the `uninitialized` part explains that  only attributes which were not used before are copied/loaded.